### DISCUSSION TITLE
MonaiLabel and Slicer error - Mac

### DISCUSSION THREAD

**Discussion by anoukstein** [Project-MONAI/MONAILabel#962] [General]

**Comment by diazandr3s** [comment]
Hi @anoukstein,

MONAI Label is tested and supported on Windows and Linux OSs (https://github.com/Project-MONAI/MONAILabel#installation). You could use Slicer on Mac and then connect to a MONAI Label server that's running on Windows or Linux.

Hope this helps.

**Reply by anoukstein** [reply]
Thank you for your prompt reply! That makes sense.
